napari/__init__.py

```diff
@@ -57,7 +57,7 @@
 # Note that importing _viewer_key_bindings is needed as the Viewer gets
 # decorated with keybindings during that process, but it is not directly needed
 # by our users and so is deleted below
-from . import _viewer_key_bindings  # noqa: F401
+# from . import _viewer_key_bindings  # noqa: F401
```

Comment:
Note for debugging I'm not loading the `Viewer` keybindings, only the `ViewerModel` ones should be functional, but they aren't - same as was reported in #1535